R/recordedplot.r

```diff
@@ -27,6 +27,7 @@ repr_png.recordedplot <- function(p,
 	#special
 	res       = getOption('repr.plot.res'),
 ...) {
+	if (!capabilities('png')) { return NULL }
```

Comment:
I assume this being FALSE would mean we couldn't generate a png - likewise for jpeg below.